### ISSUE TITLE
yolov5 model train always checks with YOLOv8n

### ISSUE THREAD

**Issue by Gitkingly** [ultralytics/ultralytics#10272]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I am running with the yolov5 sample code, I have installed ultralytics==8.0.200, when I trained the yolov5 model, it prompted this message "AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...".
An then downloaded the yolov8n.pt file automatically. This is not what I expected.

I have searched the code, it comes from https://github.com/ultralytics/ultralytics/blob/main/ultralytics/utils/checks.py.
It seems I cannot avoid this.

But I only want v5, and do my experiments base on v5, I don't want any v8.

Is there anything I missed or this is the expected behaviour?

===========================
from ultralytics import YOLO

# Load a COCO-pretrained YOLOv5n model
model = YOLO('yolov5n.pt')

# Display model information (optional)
model.info()

# Train the model on the COCO8 example dataset for 100 epochs
results = model.train(data='coco8.yaml', epochs=100, imgsz=640)

# Run inference with the YOLOv5n model on the 'bus.jpg' image
results = model('path/to/bus.jpg')
=================================

Thanks,
Jingli

**Comment by glenn-jocher** [comment]
Hi Jingli! 👋

Thanks for reaching out. It looks like you've encountered an unexpected behavior while using the Ultralytics YOLOv8 package to train a YOLOv5 model. The behavior you're experiencing, with the system defaulting to YOLOv8n checks and downloading YOLOv8n.pt, occurs because the latest Ultralytics package is designed to function primarily with YOLOv8 models.

However, if your research or experiments specifically require the YOLOv5 model, I'd recommend using the YOLOv5 repository and installation instructions. You can find the dedicated YOLOv5 GitHub repository here: https://github.com/ultralytics/yolov5. This will allow you to work directly with YOLOv5 models without involving YOLOv8 specific functionalities.

The code snippet you're working with is meant for the Ultralytics YOLOv8 environment, hence the automatic switch to YOLOv8 operations. By using the YOLOv5 repository and installation guide, you should be able to smoothly continue your experiments with YOLOv5.

Let me know if this helps or if you have any more questions! Happy to assist further. 🙂

**Comment by Gitkingly** [comment]
Thanks quickly response @glenn-jocher .  I still have questions about it. I continued to used yolov5 model with YOLOv8 package in my experiments.  It seems the predictions are correct. And I trained the yolov5 model with custom dataset, it looks like correct, too. My question is that:  Is there any other side effect except download yolov8n.pt?  It is acceptable if it only downloads yolov8n.pt while doing AMP checks.  There is a planning to use v6,v7 and v8. So I will be happy if I can use the same code to test with different models, just as you have documented here for yolov5: https://docs.ultralytics.com/models/yolov5/
